packages/@o3r/forms/README.md

```diff
@@ -12,7 +12,11 @@ This package is an [Otter Framework Module](https://github.com/AmadeusITGroup/ot
 [![Stable Version](https://img.shields.io/npm/v/@o3r/forms?style=for-the-badge)](https://www.npmjs.com/package/@o3r/forms)
 [![Bundle Size](https://img.shields.io/bundlephobia/min/@o3r/forms?color=green&style=for-the-badge)](https://www.npmjs.com/package/@o3r/forms)
 
-This module provides utilities to enhance Angular form (asynchronous decorator, additional validator, error store...).
+This module provides utilities to enhance the build of Angular reactive forms in the Otter context. These utilities include:
```

Comment:
From what I see this entire module is related to guidelines for a product application, with container/presenter structure for components.
I think we should explain this here in the beginning and maybe add another section where to say that for small apps, be-spoke apps (I don;t know how to name them :D) this module does not bring added value on top of angular forms.
Maybe we can have a discussion inside the team to double check if something from this module can be used in any use-case.
Any thoughts @pginoux-1A @kpanot ?

Reply:
I agree with you for all the complexity brings by container / presenter.
But some features could be interesting even with as small apps 
for example if you want to handle the submission at page level and display the error messages outside the form component